model/map/src/main/java/org/keycloak/models/map/loginFailure/MapUserLoginFailureAdapter.java

```diff
@@ -49,7 +51,7 @@ public void setFailedLoginNotBefore(int notBefore) {
 
     @Override
     public int getNumFailures() {
-        return entity.getNumFailures();
+        return Optional.ofNullable(entity.getNumFailures()).orElse(0);
```

Comment:
This creates a short-lived optional rather than simple `if-else` block. Is this intended?

Reply:
I don't have any particular reason for it. Replaced with traditional condition.